app/views/govuk_publishing_components/components/docs/step_by_step_nav_header.yml

```diff
@@ -5,9 +5,7 @@ body: |
 accessibility_criteria: |
   The component is designed to go into the top of an existing content page and should not interfere with the heading structure of the page, so therefore should not contain a heading tag.
 
-  The component must:
-
-  - provide a skip link to the step by step navigation on the page, if one is present
+  An early version of the component contained a hidden skip link for keyboard and screen reader users, that jumped to the step by step navigation component in the sidebar (similiar to the 'skip to content' link at the top of all GOV.UK pages). User testing suggested that rather than helping users it confused them, so this has been removed.
```

Comment:
This is the other occurrence of referencing in the doc how the code used to be.

Reply:
One of the most important parts of the criteria is that it stops future people from making the same mistake

Reply:
I would then rather say it in a form of an acceptance criterion without solutionising it, e.g.
> The component must:
> - not provide a skip link to jump to the navigation component

None of the other criteria does anything like this.
The step by step navigation also has this criterion which [should not mention any technical implementation](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/accessibility_acceptance_criteria.md#dont-define-the-solution):

> be readable when only the text of the page is zoomed - achieved for the numbers and logic elements by their text being wrapped in several elements that give them a white background and ensure the text when zoomed expands to the left, not to the right, where it would obscure the step titles

Reply:
Sounds good to rewrite it, but may be good to include context of the research so people know why it did not work.